keyboards/edi/hardlight/mk2/readme.md

```diff
@@ -0,0 +1,19 @@
+# Hard Light Mark 2
+
+![Hard Light Mark 2](https://i.imgur.com/X6Katarh.png)
+
+A compact 60% keyboard designed by Fate Everywhere. Born out of the desire for a Planck with a numpad for technical work and a lack of reliance on the Elite-C.
+
+* Keyboard Maintainer: [Fate Everywhere](https://github.com/fateeverywhere)
+* Hardware Supported: Mark 2 Hard Light.
+* Hardware Availability: [Seven Store](https://store.7storm.org).
+
+Build example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?

Reply:
Nice eye, given the pain that caused me with the prototypes. Done!